test/anonymizer.test.js

```diff
@@ -0,0 +1,30 @@
+import dcmjs from '../src/index.js';
+import fs from "fs";
+
+const { DicomMessage } = dcmjs.data;
+const { cleanTags } = dcmjs.anonymizer;
+
+it("test_export", () => {
+    expect(typeof cleanTags).toEqual("function");
+});
+
+it("test_anonymization", () => {
```

Comment:
Although we wrote poor test descriptions in the past I think we should move forward with better test descriptions by actually describing the test. Adding contextual information through 'describe' blocks is a good practice too. 

@richard-viney I don't want to inherit the tech so, for now, can you please add test instructions? or code standards for testing? this could help us keep our test codebase in better shape.

@pieper Another thing I have noticed is that we don't have code coverage and I think we should have it configured + make it part of our pipeline. I'll create an issue for it later on. No worries right now. 😄

Reply:
https://github.com/dcmjs-org/dcmjs/issues/259

Reply:
💯 for code coverage and better test descriptions.

Reply:
Test instructions and best practices have been added to `README.md`.